libs/api-interfaces/src/lib/aggregate-views/view-models/resources/media-items/mime-type.enum.ts

```diff
@@ -1,4 +1,5 @@
 export enum MIMEType {
     mp3 = 'audio/mpeg',
     mp4 = 'video/mp4',
+    wav = '	audio/x-wav'
```

Comment:
There is a typo here. We can't have a leading space. It has to be exactly as in the standard.